optin/puller/tests/client/test_SymbolClient.py

```diff
@@ -268,9 +268,36 @@ async def test_can_query_transaction_statuses(server):  # pylint: disable=redefi
 	assert str(HASHES[1]) == transaction_statuses[2]['hash']
 	assert '111003' == transaction_statuses[2]['height']
 
+# endregion
+
+
+# region find_redeem_transactions
+
+def assert_message(message, transaction):
+	assert message == transaction['transaction']['message']
+
+
+async def test_can_find_redeem_transactions(server):  # pylint: disable=redefined-outer-name
+	# Arrange:
+	client = SymbolClient(server.make_url(''))
+
+	# Act:
+	transactions = await client.find_redeem_transactions(PublicKey(PUBLIC_KEYS[0]))
+
+	# Assert:
+	assert [
+		f'{server.make_url("")}/transactions/confirmed?signerPublicKey={PublicKey(PUBLIC_KEYS[0])}&embedded=true&fromHeight=2&pageSize=100'
+	] == server.mock.urls
+	assert 3 == len(transactions)
+
+	prefix = '007b20226e697341646472657373223a20224e'
```

Comment:
uppercase 😅

Reply:
so I'd have to throw additional upper in mock server (feels weird to do it just for test purposes)